osfio-export-tool/tests/test_clitool.py

```diff
@@ -321,6 +321,60 @@ def test_get_single_mock_project(self):
         assert projects[0]['metadata']['id'] == 'x'
         assert projects[0]['children'] == ['a', 'b']
 
+    def test_write_pdf_without_giving_a_folder(self):
+        projects = [
+            {
+                'metadata': {
+                    'title': 'My Project Title',
+                    'id': 'id',
+                    'url': 'https://test.osf.io/',
+                    'description': 'This is a description of the project',
+                    'date_created': datetime.datetime.fromisoformat(
+                        '2025-06-12T15:54:42.105112Z'
+                    ),
+                    'date_modified': datetime.datetime.fromisoformat(
+                        '2001-01-01T01:01:01.105112Z'
+                    ),
+                    'tags': 'tag1, tag2, tag3',
+                    'resource_type': 'na',
+                    'resource_lang': 'english',
+                    'affiliated_institutions': 'University of Manchester',
+                    'identifiers': 'N/A',
+                    'license': 'Apache 2.0',
+                    'subjects': 'sub1, sub2, sub3',
+                },
+                'contributors': [
+                    ('Pineapple Pizza', True, 'email'),
+                    ('Margarita', True, 'email'),
+                    ('Margarine', True, 'email')
+                ],
+                'files': [
+                    ('file1.txt', None, None),
+                    ('file2.txt', None, None),
+                ],
+                'funders': [],
+                'wikis': {
+                    'Home': 'hello world',
+                    'Page2': 'another page'
+                },
+                "parent": None,
+                'children': ['a']
+            }
+        ]
+        root_nodes = [0]
+        pdf_one, path_one = write_pdf(projects, root_nodes[0], '')
+        try:
+            title_one = projects[0]['metadata']['title'].replace(' ', '-')
+            date_one = pdf_one.date_printed.strftime(
+                '%Y-%m-%d %H:%M:%S %Z'
+            ).replace(' ', '-')
+            filename = f'{title_one}-{date_one}.pdf'
+            assert filename in os.listdir(os.getcwd())
+        except Exception:
+            raise AssertionError('Unable to create file in current directory.')
```

Comment:
Could this other Errors, not AssertionError?  What if something breaks in the 3 lines above the assert. 
     

       title_one = projects[0]['metadata']['title'].replace(' ', '-')
        date_one = pdf_one.date_printed.strftime(
            '%Y-%m-%d %H:%M:%S %Z'
        ).replace(' ', '-')
        filename = f'{title_one}-{date_one}.pdf'

       try:
            assert filename in os.listdir(os.getcwd())
        except Exception:
            raise AssertionError('Unable to create file in current directory.')
```